src/main/java/com/everis/control/CarService.java

```diff
@@ -0,0 +1,62 @@
+package com.everis.control;
+
+
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+
+
+public class CarService {
+
+    @PersistenceContext(unitName = "car-unit")
```

Comment:
To inject the persistence context using the annotation you're using this class must be a managed bean.
Please annotate this class with @Stateless.

https://www.baeldung.com/ejb-session-beans